submission/programs/adae.R

```diff
@@ -108,7 +108,7 @@ adae0 <- ae %>%
     end_date = AENDT,
     out_unit = "days"
   ) %>%
-  mutate(ADURU = str_replace(ADURU, "DAYS", "DAY")) %>%
```

Comment:
we can just remove line 111 since the value stays the same

Reply:
Thanks, @kaz462 .  Makes sense. Updated and pushed adae.R with this removal.